core/modules/css/SConscript

```diff
@@ -8,20 +8,15 @@ Import('standardModule')
 pythonInc = env['PYTHON_INC_DIR']
 cppPath = env['CPPPATH'] + [pythonInc]
 
-# Use special C++ file suffix to avoid mix with standardModule()
-# Python module is generated in python/ directory for standardModule()
-swigged = env.SharedObject(source="cssLib.i",
-                           SWIGFLAGS=['-python', '-c++', '-Iinclude'],
-                           SWIGPATH=['..', pythonInc],
-                           CPPPATH=cppPath,
-                           SWIGCXXFILESUFFIX='_wrap.c++',
-                           SWIGOUTDIR=env.Dir("python"))
+# pybind11 Python wrapper goes to a separate library, cc file is also
+# excluded from standardModule()
+# pybind11 makes too many warnings when built with -pedantic
```

Comment:
Here is what appears with `-pedantic` enabled:
```
Compiling shared object build/css/cssPythonWrapper.os
In file included from /u2/salnikov/lsstsw/stack/Linux64/pybind11/master-g6f4b1f770b/include/pybind11/cast.h:13:0,
                 from /u2/salnikov/lsstsw/stack/Linux64/pybind11/master-g6f4b1f770b/include/pybind11/attr.h:13,
                 from /u2/salnikov/lsstsw/stack/Linux64/pybind11/master-g6f4b1f770b/include/pybind11/pybind11.h:36,
                 from core/modules/css/cssPythonWrapper.cc:28:
/u2/salnikov/lsstsw/stack/Linux64/pybind11/master-g6f4b1f770b/include/pybind11/pytypes.h: In lambda function:
/u2/salnikov/lsstsw/stack/Linux64/pybind11/master-g6f4b1f770b/include/pybind11/pytypes.h:1023:89: warning: ISO C++ forbids casting between pointer-to-function and pointer-to-object [-Wpedantic]
             auto destructor = reinterpret_cast<void (*)(void *)>(PyCapsule_GetContext(o));
                                                                                         ^
/u2/salnikov/lsstsw/stack/Linux64/pybind11/master-g6f4b1f770b/include/pybind11/pytypes.h: In constructor 'pybind11::capsule::capsule(const void*, void (*)(void*))':
/u2/salnikov/lsstsw/stack/Linux64/pybind11/master-g6f4b1f770b/include/pybind11/pytypes.h:1031:50: warning: ISO C++ forbids casting between pointer-to-function and pointer-to-object [-Wpedantic]
         if (PyCapsule_SetContext(m_ptr, (void *) destructor) != 0)
                                                  ^
/u2/salnikov/lsstsw/stack/Linux64/pybind11/master-g6f4b1f770b/include/pybind11/pytypes.h: In constructor 'pybind11::capsule::capsule(void (*)())':
/u2/salnikov/lsstsw/stack/Linux64/pybind11/master-g6f4b1f770b/include/pybind11/pytypes.h:1036:66: warning: ISO C++ forbids casting between pointer-to-function and pointer-to-object [-Wpedantic]
         m_ptr = PyCapsule_New(reinterpret_cast<void *>(destructor), nullptr, [](PyObject *o) {
                                                                  ^
/u2/salnikov/lsstsw/stack/Linux64/pybind11/master-g6f4b1f770b/include/pybind11/pytypes.h: In lambda function:
/u2/salnikov/lsstsw/stack/Linux64/pybind11/master-g6f4b1f770b/include/pybind11/pytypes.h:1037:92: warning: ISO C++ forbids casting between pointer-to-function and pointer-to-object [-Wpedantic]
             auto destructor = reinterpret_cast<void (*)()>(PyCapsule_GetPointer(o, nullptr));
                                                                                            ^
```
Would be nice to have upstream fixed, indeed.